main.go

```diff
@@ -67,14 +70,30 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+func runPrometheusServer(metricsAddr string) {
+	setupLog.Info("Entered runPrometheusServer")
+	mux := restful.NewContainer()
```

Comment:
Could you use the `http.ServeMux` structure from `net/http` standard library, instead of importing `github.com/emicklei/go-restful` package?

It should be enough to just replace this line with:
```
mux := http.NewServeMux()
```

Reply:
yes definitely.